web/src/routes.js

```diff
@@ -1,15 +1,15 @@
-import Activities from './containers/Activities';
+import ActivitiesPage from './components/ActivitiesPage';
 import Hello from './components/Hello';
 import NoMatch from './components/NoMatch';
-import Login from './containers/Login';
 import PoC from './components/PoC';
+import Login from './containers/Login';
 
 const routes = [
   { path: '/', component: PoC, exact: true, nonPrivate: true },
   { path: '/login', component: Login, nonPrivate: true },
   { path: '/hello', component: Hello, nonPrivate: true },
   { path: '/poc', component: PoC, nonPrivate: true },
-  { path: '/activities', component: Activities, nonPrivate: true },
+  { path: '/activities', component: ActivitiesPage, nonPrivate: true },
```

Comment:
So is the `ActivitiesPage` component just a way to view the activity components standalone?